client/package-lock.json

```diff
@@ -9,28 +9,28 @@
       "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.5.5.tgz",
```

Comment:
Bad practice to push package lock json, since it gets built on each run for each team member